Assets/MRTK/SDK/Editor/Inspectors/UX/InteractiveElement/StateSelectionMenu.cs

```diff
@@ -70,10 +72,14 @@ public void CreateStateSelectionMenu(BaseInteractiveElement instance, GenericMen
                     // Far Interaction States
                     AddStateToMenu(statesMenu, Far + "/" + stateName, stateName);
                 }
-                else if (!stateName.Contains(Far) && !stateName.Contains(Near))
+                else if (stateName == focusStateName)
                 {
-                    // Both Near and Far Interaction States
-                    AddStateToMenu(statesMenu, Both + "/" + stateName, stateName);
+                    // Focus is a special case state because it supports both near and far interaction
+                    AddStateToMenu(statesMenu, NearAndFar + "/" + stateName, stateName);
+                }
+                else
+                {
+                    AddStateToMenu(statesMenu, Other + "/" + stateName, stateName);
```

Comment:
This is how the current states are sorted in the StateSelectionMenu.  

![StatesMenu](https://user-images.githubusercontent.com/53493796/101193836-fd7b6b00-3611-11eb-8010-f46d70019c16.gif)

@davidkline-ms mentioned that a user might not think to look in the `Other` category for the Clicked and ToggleOn/Off states, which is true.  We could add a different label, but these states do not fall into the `Near`, `Far`, or `NearAndFar` category because they can potentially fall into any of those categories depending on the usage.  

However by default, the Clicked and ToggleOn/Off states are set by a far interaction selection.